pkg/machinepool/machinepool_test.go

```diff
@@ -26,6 +28,112 @@ var _ = Describe("Machinepool and nodepool", func() {
 				Value(cmv1.UpgradePolicyStateValueScheduled)).
 				NextRun(date)
 		})
+		It("editAutoscaling should equal nil if nothing is changed", func() {
```

Comment:
Tests moved here, separated by machinepool & nodepool. Taken from the original `cmd/edit/machinepool` tests, and modified to fit the refactor